src/python_testing/TC_BOOLCFG_3_1.py

```diff
@@ -125,10 +127,13 @@ async def test_TC_BOOLCFG_3_1(self):
 
         self.step(7)
         if attributes.DefaultSensitivityLevel.attribute_id in attribute_list:
-            selected_non_default_level = choice([i for i in range(numberOfSupportedLevels) if i not in [default_level]])
-            logging.info(f"Write non-default sensitivity level ({selected_non_default_level}) to CurrentSensitivityLevel)")
-            result = await self.default_controller.WriteAttribute(self.dut_node_id, [(endpoint, attributes.CurrentSensitivityLevel(selected_non_default_level))])
-            asserts.assert_equal(result[0].Status, Status.Success, "CurrentSensitivityLevel write failed")
+            if numberOfSupportedLevels > 1:
```

Comment:
If is not needed. `SupportedSensitivityLevels` always needs to be > 1 if SENSLVL feature is supported. I would recommend instead (if not already done) to check earlier that SupportedSensitivityLevels > 1 since it's mandatory. THen the if can go away